scripts/python/tao_build.py

```diff
@@ -409,13 +409,14 @@ def bazel_test(target, flag=""):
                 flag += ' --config=disc_mkldnn'
             if args.platform_alibaba:
                 flag += " --config=platform_alibaba"
+            flag += ' --test_tag_filters=-gpu'
```

Comment:
So we been executing gpu tests on CPU ...

Reply:
tests with gpu-only-*  prefix will be not be executed on cpu before. maybe the behavior has been changed during rebasing or de-coupling.